normal_operation.md

```diff
@@ -4,67 +4,42 @@ css_id: normal_operation
 
-Finally, the OEM SHOULD update the inventory database, so that the director repository is able to instruct appropriate ECUs on all affected vehicles on how to install these updated images.
+Second, the OEM SHOULD test the updated metadata and images on reserved vehicles, before releasing them to all vehicles in circulation, so that it can verify whether these images work as intended. To do so, it MAY instruct the Director repository to first install the updated images on these reserved vehicles.
 
-## Backup and garbage collection for the image repository
+Finally, the OEM SHOULD update the inventory database, so that the Director repository is able to instruct appropriate ECUs on all affected vehicles on how to install these updated images.
 
-#### TODO Add link from last sentence in this section to the Delta update strategies subsection of the Customizations section.
+## Backup and garbage collection for the Image repository
 
-The OEM SHOULD regularly perform backup and garbage collection of the metadata and images on the image repository. This is done so that the OEM is able to safely recover from a repository compromise, and ensure that the repository has enough storage space. To do so, an OEM MAY use either the following steps, or its own corporate backup and garbage collection policy.
+The OEM SHOULD regularly perform backup and garbage collection of the metadata and images on the Image repository. This is done to ensure the OEM is able to safely recover from a repository compromise, and that the repository continues to have sufficient storage space. To do so, an OEM MAY use either the following steps, or its own corporate backup and garbage collection policy.
 
-First, an automated process SHOULD store every file on the image repository, as well as its cryptographic hash on a separate, offline system. This automated process SHOULD also store a copy of the inventory database from the director repository on this offline system. This allows administrators to detect and recover from a repository compromise.
+First, an automated process SHOULD store every file on the Image repository, as well as its cryptographic hash on a separate, offline system. A copy of the inventory database from the Director repository SHOULD also be stored on this offline system. This allows administrators to detect and recover from a repository compromise. 
```

Comment:
Nitpick: extra space at the end of this line.

Reply:
Sorry, I'm not seeing an extra space. I corrected the DVD reference and would like to merge it (since both you and Trishank have looked at it)

Reply:
It got removed in your last commit, so it's all good!